src/libcore/intrinsics.rs

```diff
@@ -869,11 +869,12 @@ extern "rust-intrinsic" {
     /// // the original inner type (`&i32`) to the converted inner type
     /// // (`Option<&i32>`), so read the nomicon pages linked above.
     /// let v_from_raw = unsafe {
+    ///     // Ensure the original vector is not dropped
```

Comment:
```suggestion
    ///     // Ensure the original vector is not dropped.
```
You missed the punctuation. ;)

Reply:
Sorry. I added it. Thanks again.